File: PCS-302/01-08-2024/Brute force/p1.c
#include<stdio.h>
int main()
{
	int n,i,a[100],f,j;
	printf("Enter n: ");
	scanf("%d",&n);
	printf("Enter elements in the array:-\n");
	for(i=0;i<n-1;i++)
		scanf("%d",&a[i]);
	for(i=1;i<=n;i++)
	{
		f=0;
		for(j=0;j<n-1;j++)
		{
			if(a[j]==i)
			{
				f++;
				break;
			}
		}
		if(f==0)
		{
			printf("%d is missing\n",i);
			break;
		}
	}	
	return 0;
}

File: p1.java
import java.util.*;
public class p1
{
	public static void main(String args[])
	{
		int n1,n2,m1,m2;
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter rows and columns of first array: ");
		n1 = sc.nextInt();
		m1 = sc.nextInt();
		int a[][] = new int[n1][m1];
		
		System.out.println("Input elements in the array:-");
		for(int i=0;i<n1;i++)
		{
			for(int j=0;j<m1;j++)
				a[i][j] = sc.nextInt();
		}
		
		System.out.print("Enter rows and columns of second array: ");
		n2 = sc.nextInt();
		m2 = sc.nextInt();
		int b[][] = new int[n2][m2];
		
		System.out.println("Input elements in the array:-");
		for(int i=0;i<n2;i++)
		{
			for(int j=0;j<m2;j++)
				b[i][j] = sc.nextInt();
		}
		
		if(m1==n2)
		{
			int c[][] = new int[n1][m2];
			
		}
		else
			System.out.println("The array cant be multiplid");

		
		System.out.println("The array is:-");
		for(int i=0;i<n1;i++)
		{
			for(int j=0;j<m1;j++)
				System.out.print(a[i][j]+" ");
			System.out.println();
		}						
	}
}
